### PR TITLE
Resolve Issue #49

### PR DESCRIPTION
**Pull request by tariksouabny**
*Made on branch `dev`*

Added list of parameters in docstrings of all data fetching functions

### CODE REVIEW

**Review by Sam-damn — CHANGES_REQUESTED** [review]
Looks great to me , however i noticed you missed two methods which also need the parameter list docstrings `get_specific_parameter` and `get_coordinate_air`

**Review by Milind220 — APPROVED** [review]
Awesome! adding that in now!

EDIT: I didn't see the request for changes above. This will be merged after this is addressed.

**Comment by Milind220** [comment]
@tariksouabny Hey, please address the changes that were mentioned above

**Comment by tariksouabny** [comment]
Got it! Added the rest of the lists in the docstrings of `get_specific_parameter` and `get_coordinate_air`.

File: src/ozone/ozone.py [hunk 1]
```diff
@@ -315,6 +315,8 @@ def get_coordinate_air(
                 append the data to.
             params (List[str], optional): A list of parameters to get data for.
             Gets all parameters by default.
+								Choose from the following values:
+                    [ "aqi", "pm25", "pm10", "o3", "co", "no2", "so2", "dew", "h", "p",
```
Comment (Sam-damn) [inline src/ozone/ozone.py:319]: @tariksouabny  thanks for addressing the requests,
the new changes you made however seem to be missing few parameters which need to be added  
```suggestion
								Choose from the following values:
                    [ "aqi", "pm25", "pm10", "o3", "co", "no2", "so2", "dew", "h", "p", "t", "w", "wg" ]
```

File: src/ozone/ozone.py [hunk 2]
```diff
@@ -503,6 +513,8 @@ def get_specific_parameter(
             city (string): A city to get the data for
             air_param (string): A string containing the specified air quality parameter.
             Gets all parameters by default.
+								Choose from the following values:
+                    [ "aqi", "pm25", "pm10", "o3", "co", "no2", "so2", "dew", "h", "p",]
```
Comment (Sam-damn) [inline src/ozone/ozone.py:517]: here as well
```suggestion
								Choose from the following values:
                    [ "aqi", "pm25", "pm10", "o3", "co", "no2", "so2", "dew", "h", "p", "t", "w", "wg"  ]
```

**Review by Sam-damn — CHANGES_REQUESTED** [review]

**Review by Sam-damn — APPROVED** [review]

**Comment by lahdjirayhan** [comment]
🚀

**Pull request merged**